chem/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ init """
 
-__version__ = '1.2.0'
+__version__ = '1.2.1'
```

Comment:
```suggestion
__version__ = '1.3.0'
```

We should do a minor version bump for this change.